atoma-proxy/docs/openapi.yml

```diff
@@ -422,8 +422,22 @@ paths:
       - bearerAuth: []
 components:
   schemas:
+    Audio:
```

Comment:
If [we currently don't support audio](https://github.com/atoma-network/atoma-proxy/pull/219#discussion_r1953112331) then we should only document it when it's available

Reply:
True, removed